tools/pythonpkg/tests/fast/test_filesystem.py

```diff
@@ -139,6 +139,7 @@ def test_when_fsspec_not_installed(self, duckdb_cursor: DuckDBPyConnection, monk
         with raises(ModuleNotFoundError):
             duckdb_cursor.register_filesystem(None)
 
+    @mark.skipif(sys.platform.startswith("win", reason="ArrowFSWrapper doesn't support Windows paths")
```

Comment:
I don't think this is the correct fix - the test supports windows, it's just that the ArrowFSWrapper requires a URL style path with forward slashes (hence the POSIX calls)